externals/kyuubi-spark-sql-engine/src/main/scala/org/apache/kyuubi/engine/spark/operation/ExecuteStatement.scala

```diff
@@ -154,8 +153,27 @@ class ExecuteStatement(
   }
 
   override def setState(newState: OperationState): Unit = {
+    setStateInt(newState)
+  }
+
+  private def setStateInt(newState: OperationState, startTime: Option[Long] = None): Unit = {
     super.setState(newState)
+    if (startTime.isDefined) {
+      this.startTime = startTime.get
```

Comment:
When the `newState` of `AbstractOperation#setState` is `running`, `startTime` will be assigned the millisecond of the current time, so I can only reassign `startTime` here.